integration/kubernetes/confidential/fixtures/pod-config-secret.yaml.in

```
@@ -0,0 +1,47 @@
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+apiVersion: v1
+kind: Pod
+metadata:
+  name: secret-test-pod-cc
+spec:
+  runtimeClassName: $RUNTIMECLASS
+  containers:
+  - name: busybox
+    image: quay.io/prometheus/busybox:latest
+    imagePullPolicy: Always
+    command:
+    - sh
+    - -c
+    - |
+      env
+      if env | grep -q "SECRET_PASSWORD=unsealed_secret"; then
+        echo "unsealed environment as expected"
+      fi
+
+      if [ -f /tmp/secret-volume/password ]; then
+        if grep -q "unsealed_secret" /tmp/secret-volume/password; then
```

Similarly to point suggestion above, could we check the whole file content with something like
```suggestion
        if [ "$(cat /tmp/secret-volume/password)" == "unsealed_secret" ]; then
```

---

done ：）